consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/IbftProtocolScheduleTest.java

```diff
@@ -96,10 +99,39 @@ public void negativeBlockRewardThrowsException() {
     final IbftConfigOptions ibftConfig = mock(IbftConfigOptions.class);
     when(ibftConfig.getMiningBeneficiary()).thenReturn(Optional.empty());
     when(ibftConfig.getBlockRewardWei()).thenReturn(arbitraryBlockReward);
+    when(ibftConfig.getEpochLength()).thenReturn(3000L);
     when(genesisConfig.getIbft2ConfigOptions()).thenReturn(ibftConfig);
 
     assertThatThrownBy(() -> IbftProtocolSchedule.create(genesisConfig))
         .isInstanceOf(IllegalArgumentException.class)
         .hasMessage("Ibft2 Block reward in config cannot be negative");
   }
+
+  @Test
```

Comment:
nit: If you want, you could look at using "ParameterizedTest" ... but otherwise, all good.